docs/tutorial/cspc/tuning/tune.md

```diff
@@ -0,0 +1,305 @@
+# Tuning cStor Pools
+
+## Introduction 
+
+CStor pool(s) can be tuned via CSPC and is the recommended way to do it. Following are list of tunables that can be applied:
+- Resource and limit for pool manager containers to ensure quality of service.
+- Toleration for pool manager pod to ensure scheduling of pool pods on tainted nodes. 
```

Comment:
I am thinking not to mention toleration for pool because it is already holding NodeSelector and it will take more precedence.

Reply:
As discussed -- if the node satisfying the node selector is itself tainted.